scripts/stack/run-stack.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -eo pipefail
+
+export CURDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+if ! which docker-compose > /dev/null; then
+    echo "Could not find 'docker-compose'. Make sure it is installed and available via your PATH"
+    exit 2
+fi
+
+pushd "$CURDIR"
+
+export SECURE_STATE_DIR="$(mktemp -d)"
+trap 'rm -rf -- "$SECURE_STATE_DIR"' EXIT
```

Comment:
What does it do?

Reply:
Ah - thanks for pointing this out... this was some old code that set up a temporary directory for saving configurations that would be automatically deleted upon the script exit... but, it's not needed anymore. I'll remove this.